shared/chat/delete-history-warning/index.tsx

```diff
@@ -8,7 +8,7 @@ type Props = {
 }
 
 const DeleteHistoryWarning = ({onCancel, onDeleteHistory}: Props) => (
-  <Kb.MaybePopup onClose={onCancel}>
+  <Kb.MaybePopup onClose={onCancel} style={styles.parent}>
```

Comment:
styles.parent doesn't exist, probably need to remove

Reply:
thanks